cdap-cli/src/main/java/co/cask/cdap/cli/command/security/GrantActionCommand.java

```diff
@@ -54,13 +54,13 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     Set<Action> actions = fromStrings(Splitter.on(",").split(arguments.get("actions")));
 
     client.grant(entity, principal, actions);
-    output.printf("Successfully granted action(s) '%s' on entity '%s' for principal '%s'\n",
+    output.printf("Successfully granted action(s) '%s' on entity '%s' to principal '%s'\n",
                   Joiner.on(",").join(actions), entity.toString(), principal);
   }
 
   @Override
   public String getPattern() {
-    return String.format("security grant entity <%s> %s <%s> %s <%s> actions <actions>",
+    return String.format("grant actions <actions> on entity <%s> to %s <%s> %s <%s>",
```

Comment:
the strings ``principal-type`` and ``principal-name`` should be removed from this command as well.